api/base/views.py

```diff
@@ -6,9 +6,53 @@
 
 @api_view(('GET',))
 def root(request, format=None):
+    """
+        Welcome to the V2 Open Science Framework API. With this API you can programatically access users,
```

Comment:
Chances are all this is going to go through a huge revision in the next couple of weeks, but I'm good with someone else looking through it.